### ISSUE TITLE
Unexpected unknown property -styled-mixin0

### ISSUE THREAD

**Issue by cihati** [kristerkari/stylelint-config-react-native-styled-components#3]
Hey there!

For

```
const ImgStyle = styled.Image`
  ${borderRadius};
  ${space};
  ${width};
  ${height};
```

I get an error like:

```
 12:3  ✖  Unexpected unknown property "-styled-mixin0"   react-native/css-property-no-unknown
 13:3  ✖  Unexpected unknown property "-styled-mixin1"   react-native/css-property-no-unknown
 14:3  ✖  Unexpected unknown property "-styled-mixin2"   react-native/css-property-no-unknown
 15:3  ✖  Unexpected unknown property "-styled-mixin3"   react-native/css-property-no-unknown
```

Any ideas @kristerkari ?

Many thanks = )

**Comment by kristerkari** [comment]
Thanks @cihati!

The `react-native/css-property-no-unknown` rule is separate from styled components, and it seems that styled components is passing it some special property names.

I think that I need to make a fix to the rule, so that it ignores property names that start with `-styled-`.

**Comment by cihati** [comment]
@kristerkari thanks for the absurdly fast reply! = )

**Comment by kristerkari** [comment]
Fixed in stylelint-react-native v2.1.1:
https://github.com/kristerkari/stylelint-react-native/releases/tag/2.1.1

Make sure that you update the version in your package.json file.
